_data/rules.yml

```diff
@@ -842,6 +842,21 @@ groups:
                 query: "increase(redis_rejected_connections_total[1m]) > 0"
                 severity: critical
 
+      - name: Meilisearch
+        exporters:
+          - name: Embedded exporter
+            slug: embedded-exporter
+            doc_url: https://github.com/orgs/meilisearch/discussions/625
+            rules:                
+              - name: Meilisearch index is empty
+                description: Meilisearch instance is down
+                query: 'meilisearch_index_docs_count{index="$Index"} == 0'
```

Comment:
i think `{index="$Index"}` is not expected here

Reply:
the filter in the dashboard for grafana we use is configured this way: `meilisearch_index_docs_count{index="$Index"}` , do you think there is a better way to put it?

the last action failed due to the single `=` before the 0.

Reply:
I'm pretty sure that the `$Index` is a variable injected by Grafana.

If you want to match every indices, I suppose the `meilisearch_index_docs_count == 0` would work.

(I cannot test)

Reply:
uuuuh, youre right @samber 

We se that variable and use it in explore, but without the label will it be more general and clean!
thanks, i wanted to be more precise but that is good enough for a standard alert. :3.